code/src/app/dashboard/editcar/page.tsx

```diff
@@ -16,7 +16,10 @@ export default function page() {
           color="Red"
           seats={5}
           condition="New"
-          oilChange="2022-01-01"
+          mileageLastOilChange={20000}
+          mileageLastTireChange={30000}
+          dateNextOilChange="2022-02-02"
+          dateNextTireChange="2022-03-03"
```

Comment:
The changes made for this component is just to add the fields required. This page still needs to be implemented to retrieve car information from the DB.